solid/solid-file-client/index.html

```diff
@@ -9,7 +9,7 @@
     </head>
 
     <body>
-        <h1>Solid-File-Client Hello World</h1>
+        <h1>Solid Hello World</h1>
```

Comment:
I do not understand why you changed the name here. When going around the pages You are not sure to know where you are except reading the page content.

```suggestion
        <h1>Solid-file-client Hello World</h1>
```

Reply:
I thought the name was too long, but ok I agree it's not a great idea. I added the names back in parentheses.